### DISCUSSION TITLE
Search fields search for partial names

### DISCUSSION THREAD

**Discussion by acrispim** [Kong/kong-manager#153] [Q&A] [answered]
Can you tell me if it is possible to search for parts of the names in the search fields? For example, I have almost 50 thousand consumers and dozens of services, and I cannot search by passing partial names, the search field asks for the exact name. Is there any syntax that helps with the search?

I'm using Kong Manager OSS 3.5.0.

**Comment by Leopoldthecoder** [comment, marked as answer]
Hi @acrispim, it is not possible to search for partial names in Kong Manager OSS. Only exact names or exact entity IDs are supported. Searching for partial names is supported in the enterprise edition.

**Comment by msartor92** [comment]
seriously? this behavior made kongmanager unusable for most part of us
